### PR TITLE
Reddit scraper

### LINKED ISSUE #6: Connect SQLAlchemy to Plotly

### PR DESCRIPTION
**Pull request by timothytancy**
Closes #6, leaving BigInteger id as 64bit

### CODE REVIEW

File: utils/reddit.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import os
+
+import nltk
+import pandas as pd
+import praw
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+
+# retrieve env vars
+CLIENT_ID = os.environ.get("REDDIT_CLIENT_ID")
```
Comment (NicholasGoh) [inline utils/reddit.py]: @majulahsingapuri 

is it better if we do:

```python
from config import config
CLIENT_ID = config.reddit_client_id
CLIENT_SECRET = config.reddit_client_secret
USER_AGENT = config.reddit_user_agent
```

Comment (timothytancy) [reply utils/reddit.py]: Oh yes, must have submitted an old snippet there by mistake

**Review by NicholasGoh — COMMENTED** [review]
looks good

**Review by NicholasGoh — APPROVED** [review]

File: utils/services.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    session = Session()
+
+    # Inser the dataframe into the database in one bulk
+    conn.execute(table.insert(), write_list)
+
+    # Commit the changes
+    session.commit()
+
+    # Close the session
+    session.close()
```
Comment (majulahsingapuri) [inline utils/services.py:26]: I guess this is ok as well but `Session` comes as a context manager as well so for readability/ensuring that the connection is closed we should use that instead

**Review by majulahsingapuri — COMMENTED** [review]
Looks good from me too

**Pull request merged**